guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc

```diff
@@ -30,7 +30,7 @@ ifdef::client-content-apt[]
 * On {Debian} hosts:
 +
 ----
-# apt-get install katello-pull-transport-migrate
+# apt install katello-pull-transport-migrate
```

Comment:
```suggestion
# {client-package-install-deb} install katello-pull-transport-migrate
```

Please also add `[options="nowrap", subs="+quotes,verbatim,attributes"]` above the code block.